deployability/modules/testing/tests/test_manager/__init__.py

```diff
@@ -0,0 +1,2 @@
+from ..helpers.manager import WazuhManager
+from ..helpers.generic import HostConfiguration, CheckFiles, HostInformation, GeneralComponentActions
```

Comment:
Add newline at the end of the file
```suggestion
from ..helpers.generic import HostConfiguration, CheckFiles, HostInformation, GeneralComponentActions

```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5112/commits/dd818b765d551ab2c6431656986ec60cea05225c